Cargo.toml

```
@@ -60,8 +60,8 @@ toml_edit = { version = "0.22", features = ["serde"] }
 symlink = "0.1"
 serde_json = { version = "1.0", features = ["preserve_order"] }
 serde = { version = "1.0", features = ["derive"] }
-zombienet-sdk = "0.2.7"
-zombienet-support = "0.2.7"
+zombienet-sdk = { git = "https://github.com/paritytech/zombienet-sdk", branch = "fix-stash-gen-key"}
+zombienet-support = { git = "https://github.com/paritytech/zombienet-sdk", branch = "fix-stash-gen-key"}
```

```suggestion
zombienet-sdk = "0.2.10"
zombienet-support = "0.2.10"
```

Also, which types are you using from `support` (since those should be re-exported by the sdk)?

---

Got it, `LocalFileSystem` can be exported from the `zombienet-sdk`.